apps/rush-lib/src/utilities/LastInstallFlag.ts

```diff
@@ -0,0 +1,58 @@
+import * as path from 'path';
+import * as fsx from 'fs-extra';
+import * as _ from 'lodash';
+
+export const LAST_INSTALL_FLAG_FILE_NAME: string = 'last-install.flag';
+
+/**
+ * A helper class for managing last-install flags, which are persistent and
+ * indicate that something installed in the folder was sucessfully completed.
+ * It also compares state, so that if something like the Node JS version has changed,
+ * it can invalidate the last install.
+ * @internal
+ */
+export class LastInstallFlag {
```

Comment:
>LastInstallFlag [](start = 13, length = 15)

I feel like the class name should have the word "file' in it, since that's a pretty important aspect of this API

Reply:
We can fix this later if we decide to generalize this API

---
In reply to: [171727085](https://github.com/Microsoft/web-build-tools/pull/546#discussion_r171727085) [](ancestors = 171727085)